include/psa/crypto_values.h

```diff
@@ -1094,6 +1185,28 @@
     PSA_ALG_AEAD_WITH_TAG_LENGTH(ref, 0) ?                               \
     ref :
 
+/** Macro to build an AEAD minimum-tag-length algorithm policy.
+ *
+ * A mininimum-tag-length AEAD algorithm policy contains all AEAD algorithms
+ * sharing the same base algorithm, and where the tag length of the specific
+ * algorithm is equal to or larger then the policy's minimum tag length.
+ *
+ * \param aead_alg      An AEAD algorithm identifier (value of type
+ *                      #psa_algorithm_t such that #PSA_ALG_IS_AEAD(\p alg)
+ *                      is true).
+ * \param tag_length    Desired minimum length of the authentication tag in
+ *                      bytes.
+ *
+ * \return              The corresponding AEAD algorithm policy with the
+ *                      specified minimum length.
+ * \return              Unspecified if \p alg is not a supported
+ *                      AEAD algorithm or if \p tag_length is not valid
+ *                      for the specified AEAD algorithm.
+ */
+#define PSA_ALG_AEAD_POLICY_WITH_MINIMUM_TAG_LENGTH(aead_alg, tag_length) \
```

Comment:
Regarding naming: `PSA_ALG_AEAD_WITH_TAG_LENGTH` was renamed to `PSA_ALG_AEAD_WITH_SHORTENED_TAG` in the final release of the PSA specification. See https://github.com/ARMmbed/mbedtls/pull/3967

Reply:
Should that work be part of this PR?

Reply:
Not sure what you mean by “this work”, but the names for the new policies should be chosen based on the 1.0.0 = post-3967 macro names, not based on the beta = pre-3967 names.